caffe2/CMakeLists.txt

```diff
@@ -509,6 +509,7 @@ if (NOT INTERN_BUILD_MOBILE OR NOT BUILD_CAFFE2_MOBILE)
       ${TORCH_SRC_DIR}/csrc/api/src/jit.cpp
       ${TORCH_SRC_DIR}/csrc/api/src/nn/init.cpp
       ${TORCH_SRC_DIR}/csrc/api/src/nn/module.cpp
+      ${TORCH_SRC_DIR}/csrc/api/src/nn/modules/avgpool.cpp
```

Comment:
@ShahriarSS I am thinking we might need to add the `avgpool.cpp` entry into https://github.com/pytorch/pytorch/blob/13292ec3c7f47228b93dbef1ca750b66b4b8e0af/tools/build_variables.py#L178-L205 similar to the other C++ module PRs